api/models/User.js

```diff
@@ -8,6 +8,7 @@ let User;
 const userSchema = new mongoose.Schema({
 	name: { type: String, required: true },
 	email: { type: String, required: true },
+	phone: { type: String, required: true },
```

Comment:
Let's allow a user to not put a phone number in. Remove `required`, and add a default value of the empty string.